src/pages/Onboarding.tsx

```diff
@@ -20,20 +22,21 @@ type OnboardingScreen = {
 
 const Onboarding = () => {
     const { t } = useTranslation();
-
-    const navigate = useNavigate();
-
-    const [activeOnboardingScreen, setActiveOnboardingScreen] = useState<number>(0);
+    const [os, setOs] = useState<string>('default');
```

Comment:
Since this is being used in 3 files now, I'll do a refactor for when everything gets merged.